app/config/services.yml

```diff
@@ -4,6 +4,15 @@ parameters:
     #parameter_name: value
 
 services:
-    #service_name:
-    #    class: AppBundle\Directory\ClassName
-    #    arguments: ['@another_service_name', 'plain_value', '%parameter_name%']
+    _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+    AppBundle\:
+        resource: '../../src/AppBundle/{Command,Form,EventSubscriber,Twig,Voter}'
```

Comment:
I prefer keeping the relative path too as everyone knows this syntax so a newcomer can understand it while `%kernel.project_dir%` is specific to symfony.